src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -183,4 +217,31 @@ export class LocalGameWrapperComponent extends GameWrapper<string> implements Af
             await this.gameComponent.showLastMove(move);
         }
     }
+    public override async getConfig(): Promise<RulesConfig> {
+        // Linter seem to think that the unscubscription line can be reached before the subscription
```

Comment:
unscub? :D
sinon c'quoi le problème avec le lint? Vaut mieux rajouter un ignore que faire un truc weird non ?

Reply:
Nah j'aime pas laissé des ignore à la ramasse, surtout qu'un ignore mit pour rien se plaint pas d'être là

Reply:
C'est quoi l'erreur du linter exactement ? Voir s'il y a moyen de configurer pour éviter de devoir écrire du code crado pour le contourner.
Il y a pas un moyen plus propre d'attendre la prochaine valeur d'un observable btw ? Me semble qu'on a utilisé ça autre part, notamment pour les users, mais pas sûr. Genre rxjs a "lastValueFrom" par exemple
> Converts an observable to a promise by subscribing to the observable, waiting for it to complete, and resolving the returned promise with the last value from the observed stream.

Reply:
"Variable 'subcription' is used before being assigned."

Ca me semble suffisemment propre comme ça, vraiment, j'ai essayé plein de trucs sans succès.

Reply:
laissons en non résolu et j'essayerai un coup également avant le merge, car du crado "suffisamment propre" si on peut faire du vrai propre, c'est pas idéal :p

Reply:
Note: j'ai checké, mais ce commentaire n'a pas de sens ici, il explique un problème qui n'est pas lié au code correspondant ("Linter seem to think that the unscubscription line can be reached before the subscription" -> bah non le linter se plaint de rien dans le code présent).
C'était quoi le code qui posait problème pour le linter ?

Reply:
Donc : je dirais qu'on est bon avec ce code, faut juste mettre à jour le commentaire. En gros, on veut attendre la prochaine valeur non nulle de la config. (Me semble qu'on a fait similaire à d'autres endroits, si c'est un pattern qui revient on pourrait faire un util qui donne une promise qui resolve quand une valeur observable matche une certaine condition)